docs/docker_walkthrough.rst

```diff
@@ -0,0 +1,539 @@
+.. _docker_walkthrough:
+
+Docker Walkthrough
+==================
+
+This walkthrough provides a tutorial on how to set up Rasa Core, Rasa NLU,
+an Action Server, and Duckling with Docker containers. The tutorial starts from
+scratch. Hence, all you need is a bit of familiarity with your command line.
+
+Let's get started!
+
+.. contents::
+
+1. Setup
+--------
+
+In order to get going we need only two things installed.
+
+    - A text editor of your choice
+    - Docker
+
+If you do not have installed `Docker <https://www.docker.com/>`_ so far, you
+have to install it, too. If you are not sure whether it is installed execute the
+following command:
+
+  .. code-block:: bash
+
+    docker -v && docker-compose -v
+    # Docker version 18.06.1-ce, build e68fc7a
+    # docker-compose version 1.22.0, build f46880f
+
+If your output is not similar to the one above, please install Docker.
+See `this instruction page <https://docs.docker.com/install/>`_
+of the Docker documentation which describes how to install Docker on a variety
+of operating systems.
+
+Congratulation! 🚀 Now we have got everything set up to get going!
```

Comment:
The same with using 'we'. I would use 'you' everywhere because it refers to the user directly.  One more thing to this is that I would probably leave the 'Congratulations' part here because it makes it look like the most important part of the tutorial is already finished  :)